cmd/gophKeeperClient/main.go

```diff
@@ -0,0 +1,793 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const CommandSet = "Список команд:\n" +
+	"Регистрация нового пользователя: register\n" +
+	"Аутентификация пользователя: login\n" +
+	"Добавить новые данные: add\n" +
+	"Получеие даннных пользователя: get\n" +
+	"Удалить данные по id: delete\n" +
+	"Изменить данные по id: edit\n" +
+	"Информация о версии и дате сборки бинарного файла клиента: info"
+
+const cookieName string = "user_info"
```

Comment:
Можно сделать объявление констант более элегантным:
```suggestion
const (
	CommandSet = "Список команд:\n" +
		"Регистрация нового пользователя: register\n" +
		"Аутентификация пользователя: login\n" +
		"Добавить новые данные: add\n" +
		"Получеие даннных пользователя: get\n" +
		"Удалить данные по id: delete\n" +
		"Изменить данные по id: edit\n" +
		"Информация о версии и дате сборки бинарного файла клиента: info"

	CookieName string = "user_info"
)
```